src/middleware/sentry.js

```diff
@@ -28,7 +24,9 @@ const replaceIds = (string) => {
 const removeIdMiddleware = (event) => {
 	if (event && event.request) {
 		// eslint-disable-next-line camelcase
-		const { request: { data, url, query_string } } = event;
+		const {
+			request: { data, url, query_string },
```

Comment:
Codacy found an issue: [Identifier 'query_string' is not in camel case.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045788)